modules/monitoring/application/controllers/TacticalController.php

```diff
@@ -8,9 +8,19 @@
 use Icinga\Web\Url;
 use Icinga\Web\Widget\Tabextension\DashboardAction;
 use Icinga\Web\Widget\Tabextension\MenuAction;
+use Icinga\Web\Widget\Tabextension\OutputFormat;
 
 class TacticalController extends Controller
 {
+    /**
+     * @see ActionController::init
```

Comment:
Please don't add such boilerplate documentation. If you don't have anything more to say, (for overridden methods) just omit the documentation entirely.

Reply:
I literally just copied this method from the ListContoller char by char.
So you would rather have me take out the entire docu for init() ?

Reply:
That's how we handle this now. The code in the ListController is simply old.